README.md

```diff
@@ -8,23 +8,26 @@ This dataset is licensed under a https://creativecommons.org/licenses/by-nc/4.0/
 
 Available online at https://doi.org/10.5281/zenodo.1154518
 
+Conceptlists in Concepticon:
+- [Lieberherr-2017-100](http://concepticon.clld.org/contributions/Lieberherr-2017-100)
 
 ## Statistics
 
 
-[![Build Status](https://travis-ci.org/lexibank/lieberherrkhobwa.svg?branch=master)](https://travis-ci.org/lexibank/lieberherrkhobwa)
+[![Build Status](https://travis-ci.org/None.svg?branch=master)](https://travis-ci.org/None)
```

Comment:
we're still getting None here

Reply:
Just an oversight from my part, already fixed.

Reply:
https://github.com/lexibank/lieberherrkhobwa/pull/16/commits/74148fe8baa5147334f11fefc877f2b5ef598e5a